CHANGELOG.md

```diff
@@ -41,8 +41,7 @@ _This release is scheduled to be released on 2024-01-01._
 - Fix #3256 filter out bad results from rrule.between
 - Fix calendar events sometimes not respecting deleted events (#3250)
 - Fix electron loadurl locally on Windows when address "0.0.0.0" (#2550)
-- Fix updatanotification (update_helper.js): catch error if reponse is not an JSON format (check PM2)
-- Fix missing typeof in calendar module
```

Comment:
The deletion of line 45 was not intentional, was it?

Reply:
maybe a oversight when i merged develop into it. will merge develop again now that other PRs got merged and re-add it